internal/models/changelog.go

```diff
@@ -1,16 +1,9 @@
 package models
 
 // ChangelogEntry - JSON for individual rule from changlog table
-type ChangelogEntry struct {
-	ID           int    `json:"id"`
-	ResponseCode string `json:"response_code"`
-	EnhancedCode string `json:"enhanced_code"`
-	Regex        string `json:"regex"`
-	Priority     int    `json:"priority"`
-	Description  string `json:"description"`
-	BounceAction string `json:"bounce_action"`
-	UserID       string `json:"userid"`
-	CreatedAt    string `json:"createdat"`
-	Comment      string `json:"comment"`
-	
-}
\ No newline at end of file
+// type ChangelogEntry struct {

```

Comment:
We should remove dead code. We don't want to commit commented/unused code to master